code/datums/mind.dm

```diff
@@ -59,6 +59,8 @@ datum/mind
 
 	var/show_respawn_prompts = TRUE
 
+	var/artifacts_activated = 0
```

Comment:
Consider just putting this on the objective datum unless there's a reason we'd want to track this otherwise?